WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -26,17 +27,20 @@ public class Program
 		// This is only needed to pass CrashReporter to AppMainAsync otherwise it could be a local variable in Main().
 		private static CrashReporter CrashReporter;
 
+		private static TerminateService TerminateService;
+
 		// Initialization code. Don't use any Avalonia, third-party APIs or any
 		// SynchronizationContext-reliant code before AppMain is called: things aren't initialized
 		// yet and stuff might break.
 		public static void Main(string[] args)
 		{
 			bool runGui = false;
 			Exception? appException = null;
+			CrashReporter = new CrashReporter();
 
 			try
 			{
-				CrashReporter = new CrashReporter();
+				TerminateService = new TerminateService(TerminateApplicationAsync);
```

Comment:
I would say this belongs under `CrashReporter`. When something is in `try` block, one expects that it may throw but it looks to me that `TerminateService` is not supposed to throw and it never will.

Reply:
ACK

Reply:
I agree, moved.